native/explorer/src/cloud_writer.rs

```diff
@@ -29,6 +29,8 @@ impl CloudWriter {
     /// which bridges the sync writing process with the async ObjectStore multipart uploading.
     pub fn new(object_store: Box<dyn ObjectStore>, path: Path) -> Self {
         let runtime = tokio::runtime::Builder::new_current_thread()
+            .enable_time()
+            .enable_io()
```

Comment:
Great work! Apologies for missing these when I copied the example over from the Polars repository 🙈

Reply:
No problem! :D
I couldn't understand why they are needed, but without them, we get panics.